### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#692] [Galaxy Trivia] [answered]
<img width="275" alt="image" src="https://user-images.githubusercontent.com/5120290/236582340-78d78133-4151-43ae-b2be-16d3c87e4101.png">

## Objective / Rules

- Complete the grid using all of the digits 1 to 9.
- When completed all of the sums must be correct.
- The sums are solved strictly left to right, top to bottom.
- The normal order of mathematical operations is ignored.
- For example 2 + 5 x 9 is calculated as (2 + 5) x 9 = 63.
- There are no ÷ 1 in the puzzle.
- There are no x 1 in the puzzle (although there might be 1 x).
- At no point will any calculation go below zero, or fractional.

**Comment by peterbaccaro** [comment, marked as answer]
![puzz](https://user-images.githubusercontent.com/74666079/236677902-6bc3491c-ce26-4cea-a5b3-8e1f9f4e808a.png)

**Reply by ParanoidUser** [reply]
Bravo! 👏👏
